target/linux/ath79/dts/ar9344_devolo_magic-2-wifi.dts

```diff
@@ -0,0 +1,196 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Devolo Magic 2 Wifi";
+	compatible = "devolo,magic-2-wifi", "qca,ar9344";
+
+	aliases {
+		led-boot = &plcr;
+		led-failsafe = &plcr;
+		led-running = &plcw;
+		led-upgrade = &plcr;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan: wlan {
+			label = "devolo:status:wlan";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
```

Comment:
In this case, you arbitrarily chose phy1, while phy0 has no indicator (which is also stated in the commit message).
I just want to highlight this again for other reviewers, maybe there is a way to indicate both phyX?

Reply:
I couldn't find it, and the device itself doesn't list it either: 
root@OpenWrt:/sys# cat /sys/devices/platform/leds/leds/devolo:status:wlan/trigger
none switch0 timer default-on netdev phy0rx phy0tx phy0assoc phy0radio phy0tpt phy1rx phy1tx phy1assoc phy1radio [phy1tpt]

Reply:
I also don't think there is a better solution than what you chose.
I just wanted to highlight this so it is not overlooked by another reviewer which is smarter than I am :-)